manifests/charts/istio-control/istio-discovery/templates/reader-clusterrole.yaml

```diff
@@ -21,6 +21,9 @@ rules:
   - apiGroups: ["networking.istio.io"]
     verbs: [ "get", "watch", "list" ]
     resources: [ "workloadentries" ]
+  - apiGroups: ["networking.x-k8s.io", "gateway.networking.k8s.io"]
```

Comment:
`networking.x-k8s.io` is gone, I don't think we need it anymore

Reply:
I was just copying what we had in the main ClusterRole for reading gatewayapi.